script_task.rs: Use self::MixedMessage in handle_msgs

nerith opened the pull request:
handle_msgs contains many references to variants of MixedMessage.

The function should directly reference the variants instead of
prepending `MixedMessage::`.

Fixes #9169.

<!-- Reviewable:start -->

[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/servo/servo/9179)

<!-- Reviewable:end -->

KiChjang commented:
@bors-servo r+

Thanks for doing this!

bors-servo commented:
:pushpin: Commit d75021b has been approved by `KiChjang`

<!-- @bors-servo r=KiChjang d75021b987ff433e065eea31e099f3ce3b243686 -->

bors-servo commented:
:hourglass: Testing commit d75021b with merge 8c03ea3...

bors-servo commented:
:broken_heart: Test failed - [linux-dev](http://build.servo.org/builders/linux-dev/builds/1141)

KiChjang commented:
```
./components/script/script_task.rs:864: use statement spans multiple lines
```

KiChjang commented:
@bors-servo delegate+
r=me after you've fixed tidy

bors-servo commented:
:v: @nerith can now approve this pull request

nerith committed:
script_task.rs: Use self::MixedMessage in handle_msgs

handle_msgs contains many references to variants of MixedMessage.

The function should directly reference the variants instead of
prepending `MixedMessage::`.

Fixes #9169.

nerith commented:
@bors-servo r=KiChjang

bors-servo commented:
:pushpin: Commit 0b25737 has been approved by `KiChjang`

<!-- @bors-servo r=KiChjang 0b2573711352dffd8c49364623c35164d6f18268 -->

bors-servo commented:
:hourglass: Testing commit 0b25737 with merge f3aa5b4...

bors-servo commented:
:broken_heart: Test failed - [linux-rel](http://build.servo.org/builders/linux-rel/builds/1279)

KiChjang commented:
@bors-servo retry
* #8769

bors-servo commented:
:zap: Previous build results for [android](http://build.servo.org/builders/android/builds/1144), [gonk](http://build.servo.org/builders/gonk/builds/1168), [linux-dev](http://build.servo.org/builders/linux-dev/builds/1142), [mac-dev-ref-unit](http://build.servo.org/builders/mac-dev-ref-unit/builds/1176) are reusable. Rebuilding only [linux-rel](http://build.servo.org/builders/linux-rel/builds/1279), [mac-rel-css](http://build.servo.org/builders/mac-rel-css/builds/1169), [mac-rel-wpt](http://build.servo.org/builders/mac-rel-wpt/builds/1199)...

bors-servo commented:
:broken_heart: Test failed - [linux-rel](http://build.servo.org/builders/linux-rel/builds/1280)

KiChjang commented:
@bors-servo retry
* #9181, #8769

bors-servo commented:
:zap: Previous build results for [android](http://build.servo.org/builders/android/builds/1144), [gonk](http://build.servo.org/builders/gonk/builds/1168), [linux-dev](http://build.servo.org/builders/linux-dev/builds/1142), [mac-dev-ref-unit](http://build.servo.org/builders/mac-dev-ref-unit/builds/1176), [mac-rel-css](http://build.servo.org/builders/mac-rel-css/builds/1169), [mac-rel-wpt](http://build.servo.org/builders/mac-rel-wpt/builds/1199) are reusable. Rebuilding only [linux-rel](http://build.servo.org/builders/linux-rel/builds/1280)...

bors-servo commented:
:sunny: Test successful - [android](http://build.servo.org/builders/android/builds/1144), [gonk](http://build.servo.org/builders/gonk/builds/1168), [linux-dev](http://build.servo.org/builders/linux-dev/builds/1142), [linux-rel](http://build.servo.org/builders/linux-rel/builds/1281), [mac-dev-ref-unit](http://build.servo.org/builders/mac-dev-ref-unit/builds/1176), [mac-rel-css](http://build.servo.org/builders/mac-rel-css/builds/1169), [mac-rel-wpt](http://build.servo.org/builders/mac-rel-wpt/builds/1199)
